openedx/core/djangoapps/models/course_details.py

```diff
@@ -226,6 +230,14 @@ def update_from_json(cls, course_key, jsondict, user):  # pylint: disable=too-ma
             descriptor.license = jsondict['license']
             dirty = True
 
+        if 'learning_info' in jsondict:
```

Comment:
Add coverage for this in test_course_details.py (most likely in the existing test_update_and_fetch test).